ros/src/OdometryServer.hpp

```diff
@@ -79,10 +81,6 @@ class OdometryServer : public rclcpp::Node {
     /// Global/map coordinate frame.
     std::string odom_frame_{"odom"};
     std::string base_frame_{};
-
-    /// Covariance diagonal
```

Comment:
Not something to tackle right away.  But I'd prefer if we keep this fixed covariance as an optional parameter. Similar to how we do (only in python) for the adaptive threshold. I believe many people would like to fuse the KISS ICP output and just give it a constant "weight"

Reply:
Make sense ;)